### ISSUE TITLE
Clicking on native ads from other source (i.e out side from browser), it is not redirecting to play store

### ISSUE THREAD

**Issue by nitish2429** [balzathor/UltimateBrowserProject#91]
I am using your stable version 1.4.6. When we click on banner  ads from external source (like from other app), it is not redricting to play store, it searches  url in google search. Basically it is not parsing the url of the native ads.

If we are clicking of banner ads from browser it works fine and also parsing the url of the ads and also redricting to play store.

**Comment by Thunderbottom** [comment]
I'll take a look, will fix it by tonight. 

Thanks for reporting :)

**Comment by nitish2429** [comment]
For Your Reference, here are some screen shots. 
In this i am clicking on Twitter Banner ad. 

Thnaks in advance.

1.

![screenshot_2016-06-20-10-41-50 1](https://cloud.githubusercontent.com/assets/17546525/16183841/25c05220-36d4-11e6-939f-2cc039baf43d.png)

2.

![screenshot_2016-06-20-10-42-07 1](https://cloud.githubusercontent.com/assets/17546525/16183847/32d2f6ca-36d4-11e6-92e2-02634dd43f07.png)

3.

![screenshot_2016-06-20-10-42-23 1](https://cloud.githubusercontent.com/assets/17546525/16183850/3e490120-36d4-11e6-84fa-f284a7efa61f.png)

**Comment by xdevs23** [comment]
We either aren't handling intents or maybe the adblocker is blocking the URL from being loaded.

**Comment by nitish2429** [comment]
In both cases ad block is enabled or disabled it happens same.

**Comment by Thunderbottom** [comment]
I guess there's no intent handling for ads.

**Comment by xdevs23** [comment]
Yup

**Comment by nitish2429** [comment]
any update ?

**Comment by Thunderbottom** [comment]
Too busy with other stuff, will update in a few days

**Comment by takahirom** [comment]
@Thunderbottom 
Long time no see:)
Where is current source code?(Can I access source code?)
If I can it easily , I will send a pull request .

**Comment by Thunderbottom** [comment]
@takahirom We have shifted the code base to Bitbucket, because many people have been using the exact same code with my Fabric API, so I get all their crash reports, which I didn't want. Also, I've seen many apps on the play store with just changed name and logo. So for now, me and @xdevs23 decided to go partially closed source (just for the time being, till I gather time to develop).

If you want, I can add you to the maintainers on Bitbucket.

**Comment by xdevs23** [comment]
Exactly, I will also work on UBP when I have time for it, I guess about next week.
